packages/expo/AppEntry.js

```diff
@@ -1,3 +1,4 @@
+import 'expo/build/Expo.fx';
```

Comment:
We should create a doc in the future which describes how to create a custom main entry point.